### ISSUE TITLE
Ships on same machine can't communicate without exposed ames port

### ISSUE THREAD

**Issue by patosullivan** [urbit/urbit#6357]
I upgraded mine and my wife's ships to 1.21 the other day and afterwards the two ships were not able to communicate, though they were able to communicate with other ships on the network.

I also have some moons running on the local network that could not communicate with each other or with new ships on the (urbit) network (they could communicate with ships they already knew about just fine, but not my planet or my wife's planet).

I opened up two ports on my NAT and ran the two planets with ./run -p <PORT> and the problems all went away (including the problems with the moons). I hadn't needed to do that previously.

Posted here because I'm not certain it's a vere issue since we also had an OTA this week on the same day I upgraded vere for the two planets, but I can move this to the vere repo if that is more appropriate.

**Comment by Fang-** [comment]
I don't think we have any mechanisms for local network peer discovery right now, so the best your planets can do is the usual route discovery, which goes through the sponsor/external network, and as such are subject to the usual requirements around reachability there.

We do probably want ways to tighten down these routes to local networking where possible, or do discovery on that level. Others may have better ideas of what exactly that would entail, but I suspect it would be a vere-level concern.
